src/client/activity/UserActivity.js

```diff
@@ -61,32 +51,32 @@ import UserActivityActions from './UserActivityActions';
     getAccountWithFollowingCount,
     getUserEstAccountValue,
     updateAccountHistoryFilter,
+    setInitialCurrentDisplayedActions,
   },
 )
 class UserActivity extends React.Component {
   static propTypes = {
-    location: PropTypes.shape().isRequired,
-    totalVestingShares: PropTypes.string.isRequired,
-    totalVestingFundSteem: PropTypes.string.isRequired,
-    user: PropTypes.shape().isRequired,
+    currentDisplayedActions: PropTypes.arrayOf(PropTypes.shape()),
```

Comment:
Not required props are usually below requires ones (`currentDisplayedActions` and `isCurrentUser`).